src/@next/components/molecules/OrderTable/test.tsx

```diff
@@ -129,8 +129,8 @@ describe("<OrderTable />", () => {
     expect(
       pushSpy
     ).toHaveBeenCalledWith(
-      `http://order-history/[token]?token=${token}`,
-      `/${token}`,
+      `/order-history/[token]?token=${token}`,
+      `/order-history/${token}`,
```

Comment:
Can we use paths from src/paths.ts ? So there would be no more need for changing those in the future